packages/synapse-react-client/src/components/SchemaDrivenAnnotationEditor/field/SynapseAnnotationsRJSFObjectField.tsx

```diff
@@ -30,49 +33,48 @@ export function SynapseAnnotationsRJSFObjectField<
    * @param props
    */
   useEffect(() => {
-    const { schema, formData, onChange } = props
     const newFormData: Record<string, any> = { ...formData }
-    if (schema[PROPERTIES_KEY]) {
-      Object.entries(schema[PROPERTIES_KEY]).forEach(
-        ([key, propertySchema]) => {
-          const data = newFormData[key]
+    const allProperties = getAllPropertiesInFlatObjectSchema(schema)
```

Comment:
Only changed to use new helper method which will find properties that are not in `schema.properties` (may be in `schema.definitions`, referenced by `allOf`